aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -338,34 +338,12 @@ module aptos_framework::stake {
         pool_address: address,
     }
 
-    /// Stores transaction fees assigned to validators. All fees are distributed to validators
-    /// at the end of the epoch.
+    #[deprecated]
+    /// DEPRECATED
     struct ValidatorFees has key {
```

Comment:
This we can remove in a separate transaction./ proposal, right? Once this change lands on mainnet and the framework is updated?

Reply:
we cannot remove structs with key ability